README.md

```diff
@@ -110,6 +110,54 @@ curl localhost:8080
 # Output: Hello World from a PHP HTTP function!
 ```
 
+## Run your function on Google Cloud Functions
+
+Follow the steps below to deploy to Google Cloud Functions. More information
+on function deployment is available in the
+[GCF documentation](https://cloud.google.com/functions/docs/deploying).
+
+To run your function on Cloud Functions, first you must have the [gcloud SDK][gcloud] installed and [authenticated][gcloud-auth].
+
+Additionally, you need to have a Google Cloud project ID for the
```

Comment:
This sentence should not be necessary given the previous link to the `Authorizing Cloud SDK tools` doc and using the configured gcloud project.

Reply:
Fair enough, removed